workflows/eager.nf

```diff
@@ -81,7 +81,7 @@ workflow EAGER {
 
 
     // Reference
-    fasta                = file(params.fasta, checkIfExists: true)
+    fasta                = params.fasta ? file(params.fasta, checkIfExists: true) : params.fasta_sheet ? file(params.fasta_sheet, checkIfExists: true) : []
```

Comment:
Not a huge fan of having a variable named after a parameter that sometimes does not take the parameter's values as input :/ 

In all other cases the variable is named after the parameter, and takes its value, but here it might be one or another parameter's value.
If this variable is not used much, I'd be in favour of naming it `fasta_fn` or sth to differentiate it from the parameter.

Reply:
I've changed the parameter name here and when it goes into reference indexing, but kept all mapping operations below that because there it is the path to the actual fasta file.